src/client/components/builder/LogicTab.tsx

```diff
@@ -39,6 +39,15 @@ export const LogicTab: FC = () => {
   const [nextUniqueId, setNextUniqueId] = useState<number>(1)
   const { dispatch, config } = useCheckerContext()
 
+  useEffect(() => {
+    let highestIndex = 0
+    config.operations.forEach((field) => {
```

Comment:
Could we use `operation` as the param since this is `forEach()` over `operations`?